runelite-client/src/main/java/net/runelite/client/plugins/grandexchange/GrandExchangePlugin.java

```diff
@@ -318,6 +350,52 @@ public void onGameTick(GameTick event)
 
 		log.debug("Looking up item {}", itemId);
 
+		String boughtText = "<br>Bought: ";
+		String soldText = " Sold: ";
+		String unknownText = "Unknown";
+		String timeStamp = " Last Trade: ";
+		if (config.enableGeBuySellHistory() && !geText.getText().contains(boughtText))
+		{
+			GrandExchangePriceHistory geBuySell = geBuySellHistory.get(itemId);
+			StringBuilder build = new StringBuilder();
+			build.append(geText.getText());
+
+			// Handles bought price lookup
+			build.append(boughtText);
+			if (geBuySell != null && !geBuySell.getBought().equals(0))
+			{
+				build.append(StackFormatter.formatNumber(geBuySell.getBought()));
+			}
+			else
+			{
+				build.append(unknownText);
+			}
+
+			// Handles sold price lookup
+			build.append(soldText);
```

Comment:
This would read better using commas to separate the "Bought", "Sold" and "Last Trade" items.

```
Bought: 100gp Sold: 10:00:12 Last Trade: 09:53:43
```

-->

```
Bought: 100gp, Sold: 10:00:12, Last Trade: 09:53:43
```

Reply:
The commas do help separate it well. Thanks for the suggestion!